src/util.js

```diff
@@ -620,5 +620,41 @@ export default {
    */
   removeTrailingSpaces: function(text) {
     return text.replace(/[ \t]+$/mg, "");
+  },
+
+  /**
+   * Encode input buffer using Z-Base32 encoding.
+   * See: https://tools.ietf.org/html/rfc6189#section-5.1.6
+   *
+   * @param {Uint8Array} data The binary data to encode
+   * @returns {String} Binary data encoded using Z-Base32
+   */
+  encode_zbase32: function(data) {
```

Comment:
Is snake case OK here? Some functions in this file use camelCase some snake_case.

Reply:
I know this function isn't that big, but perhaps it would be an idea to move it to a dedicated `encoding/zbase32.js` file? Since only one file needs it I don't think it needs to be in util